docs/reference/data-frames/apis/get-transform.asciidoc

```diff
@@ -51,17 +51,20 @@ see {stack-ov}/security-privileges.html[Security privileges] and
 ==== {api-query-parms-title}
 
 `allow_no_match` (Optional)::
-  (boolean) Whether to ignore if a wildcard expression matches no
-  {dataframe-transforms}. This includes `_all` string or when no transforms have
-  been specified. The default is `true`.
+  (boolean) Specifies what to do when the request contains a wildcard expression
+  and there are no {dataframe-transforms} that match. The default value is
+  `true`, which returns an empty `transforms` array in this scenario. If it's
```

Comment:
It is also if there is a partial match. If somebody supplies the patter `test-id*,missing*` and `missing*` does not match but `test-id*` does, we will return the `test-id*` matched transforms without the `missing*`. 

If the `allow_no_match` is `false`, we would through the 404, even if there is a partial match for one of the patterns.